frame/message-queue/Cargo.toml

```diff
@@ -0,0 +1,53 @@
+[package]
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+name = "pallet-message-queue"
+version = "7.0.0-dev"
+license = "Apache-2.0"
```

Comment:
Here's Apache-2.0 but the files' header seems GPL3,
Could you ensure these pallet release under Apache-2.0 ?

Reply:
Yea that is wrong. Will change. Thanks for spotting it!

Reply:
https://github.com/paritytech/substrate/pull/12895